app/jobs/application_job.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class ApplicationJob < ActiveJob::Base
+  # Automatically retry jobs that encountered a deadlock
```

Comment:
This was produced by the generator (rails generate job) when I generated a new job class, and provides potentially useful configuration and documentation. So I thought it worth keeping. 
But another argument I can think of is that it could get out of date (new versions of rails might generate it differently), so it would be better to only include it if needed.
So I agree to remove it.

Reply:
It's now been included on another PR so this can be ignored.